cloud/aws/templates/aws_oidc/variable_definitions.json

```diff
@@ -449,6 +449,12 @@
     "tfvar": true,
     "type": "string"
   },
+  "INGRESS_SG_CIDR": {
```

Comment:
question (non-blocking): Is there an order to how variables are entered in here?

Reply:
not really - I put it next to something somewhat similar but if there's a different order you think would be better, I can update it!

Reply:
nope, i think this is fine. but might be nice to alphabetize the whole list in a separate PR so it's easier to find/place things. or organize it by category. what do you think?

Reply:
Great idea, I'll ask.